### PR TITLE
Update deployment YAML used in IT

### PR DESCRIPTION
**Pull request by rishabh-11**
**What this PR does / why we need it**:
Allows to run IT in a way where controllers are run inside the cluster as containers.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement user
Updates deployment YAML used in IT when controllers are run as containers in the cluster.
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,20 +9,21 @@ IMAGE_TAG           := $(shell cat VERSION)
 PROVIDER_NAME       := Azure
 PROJECT_NAME        := gardener
 CONTROL_NAMESPACE  := default
-CONTROL_KUBECONFIG := dev/control-kubeconfig.yaml
-TARGET_KUBECONFIG  := dev/target-kubeconfig.yaml
+CONTROL_KUBECONFIG := ./dev/control-kubeconfig.yaml
+TARGET_KUBECONFIG  := ./dev/target-kubeconfig.yaml
```
Comment (himanshu-kun) [inline Makefile:13]: why are these changes required?

**Review by himanshu-kun — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (rishabh-11) [reply Makefile:13]: They are not needed, I noticed it after I made the commit, so I let it be. I'll revert it back.

**Review by himanshu-kun — APPROVED** [review]
/lgtm
